src/dualscreen.tsx

```diff
@@ -91,6 +91,10 @@ export class DualScreen extends KalturaPlayer.core.BasePlugin implements IEngine
 
   loadMedia(): void {
     this._addBindings();
+    const {
+      store: {dispatch}
+    } = this._player.ui;
+    dispatch(shell.actions.addPlayerClass(HAS_DUAL_SCREEN_PLUGIN_OVERLAY));
```

Comment:
it seems like `HAS_DUAL_SCREEN_PLUGIN_OVERLAY` classname always set to player if dual-screen plugin exist in uiConf. I think for cases when media doesn't have dual-screen layouts, ex: slides (image player) or secondary media (secondary player), classname should not be added.
Let's discuss it on Monday.